lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -1000,6 +845,104 @@ sub _resolve_workflow_for_build {
                 );
             }
         }
+        #Add gene category annotations to some output files from steps above. (e.g. determine which SNV affected genes are kinases, ion channels, etc.)
+        #AnnotateGenesByCategory - gene_category_<exome|wgs|wgs_exome>_<snv|indel>_result
+        my $annotate_genes_by_category_op = $self->annotate_genes_by_category_op($workflow);
+        $workflow->create_link(
+            source               => $converge_snv_indel_report_ops[-1],
```

Comment:
If we don't want to sort them, the `for` loop could decide to only keep track of the job for the highest quality threshold, something like:
```perl
my ($best_converge_op, $best_converge_op_quality) = (undef, -Inf);  #or whatever names
for my $i (1..@$mq) {
   ...
   if ($mq[$i] > $best_converge_op_quality) {
     $best_converge_op_quality = $mq[$i];
     $best_converge_op = $converge_snv_indel_report_op;
   }
}
```